UM/Trust.py

```diff
@@ -386,6 +393,12 @@ def signedFolderCheck(self, path: str) -> bool:
 
         try:
             json_filename = os.path.join(path, TrustBasics.getSignaturesLocalFilename())
+            storage_filename = os.path.join(path, TrustBasics.getCentralStorageFilename())
+
+            storage_json = None
+            if os.path.exists(storage_filename):
+                with open(storage_filename, "r", encoding="utf-8") as data_file:
```

Comment:
Code style: Spaces around binary operators.
```suggestion
                with open(storage_filename, "r", encoding = "utf-8") as data_file:
```